docs/workflow-dispatch.md

```diff
@@ -0,0 +1,126 @@
+# Workflow dispatch
+
+This documentation applies when the `dispatch-type` input is set to `workflow`, which instructs the action to create [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) events.
+
+To learn about the `workflow_dispatch` event, see GitHub's documentation on [manually running a workflow](https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#manually-running-a-workflow).
+
+## Action behaviour
+
+There are significant differences in the action's behaviour when using `workflow` dispatch.
+
+- When issuing a slash command with arguments, *only named arguments are accepted*. Unnamed arguments will be ignored.
+- A maximum of 10 named arguments are accepted. Additional named arguments after the first 10 will be ignored.
+- `ref` is a reserved named argument that does not count towards the maximum of 10. This is used to specify the target reference of the command. The reference can be a branch, tag, or a commit SHA. If you omit the `ref` argument, the target repository's default branch will be used.
+- A `client_payload` context cannot be sent with [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) events. The target workflow can only make use of up to 10 pre-defined inputs, the names of which must match named arguments supplied with the slash command.
```

Comment:
this paragraph perfectly describes the limitations of a `workflow_dispatch`.

what I am missing in the docs is a example/description why/when it makes sense to use `workflow_dispatch` over `repository_dispatch`. when does `workflow_dispatch` have a benefit over `repository_dispatch`?

Reply:
To be honest, I'm not sure myself yet. I wanted to add the feature and allow people to execute `workflow_dispatch` workflows with slash commands, but I'm not sure exactly why you would choose that workflow type over `repository_dispatch` if you were intending to execute it by slash command from the start. I think what is more likely is that users are going to start to create `workflow_dispatch` workflows to run manually from the UI. Then they might find this action and be interested in executing them by slash command. This feature allows them to do that without converting the workflow to `repository_dispatch`.

If additionally benefits become clear in the future, I'll update the docs. For now I hope the table at the following link will suffice.
https://github.com/peter-evans/slash-command-dispatch/tree/dev#dispatch-type

Reply:
I see, thx for this insights.